docker-compose.production.yml

```diff
@@ -52,7 +52,8 @@ services:
     environment:
       KQUEENUI_PREFERRED_URL_SCHEME: https
       KQUEEN_UI_CONFIG_FILE: config/prod.py
-      KQUEENUI_DEBUG: False
+      KQUEENUI_DEBUG: 'False'
+      # TODO: set SECRET_KEY
       KQUEENUI_SECRET_KEY: 'SecretSecretSecret123'
```

Comment:
Both secret keys in this file should be empty strings I default imho. When you don't set them in the specific deployment, application will fail to start which is better than starting with insecure SECRET_KEY, because someone forgot to override this variable.

Reply:
good point, ill change it

Reply:
Guys, we should check that key is set before the deployment, at least we can send warning log about that. What do you think?

Reply:
in case of empty key deployment will fail with verbose trace, u can check it , imo it looks good
@katyafervent

Reply:
I understand, but how many steps you should done before you got message about secret key is unset